README.md

```diff
@@ -1,4 +1,262 @@
+    to a member of the working group.
+  - At this point, the status in the RFC markdown document should be marked "Review"
+  - Build consensus and integrate feedback. RFCs that have broad support are
+    much more likely to make progress than those that don't receive any
+    comments. Feel free to reach out to the RFC assignee in particular to get
+    help identifying stakeholders and obstacles.
+  - The working group will discuss the RFC pull request, as much as possible in the
+    comment thread of the pull request itself. Offline discussion will be
+    summarized on the pull request comment thread.
+  - RFCs rarely go through this process unchanged, especially as alternatives
+    and drawbacks are shown. You can make edits, big and small, to the RFC to
+    clarify or change the design, but make changes as new commits to the pull
+    request, and leave a comment on the pull request explaining your changes.
+    Specifically, do not squash or rebase commits after they are visible on the
+    pull request.
+
+### Status: Final Call (FCP)
+
+At some point, a member of the working group will propose a motion for _"final
+comment period"_ (FCP), along with a *disposition* for the RFC (merge, close,
+    or postpone).
+
+  - This step is taken when enough of the tradeoffs have been discussed that
+  the working group is in a position to make a decision. That does not require
+  consensus amongst all participants in the RFC thread (which is usually
+  impossible). However, the argument supporting the disposition on the RFC
+  needs to have already been clearly articulated, and there should not be a
+  strong consensus *against* that position outside of the working group. Working group
+  members use their best judgment in taking this step, and the FCP itself
+  ensures there is ample time and notification for stakeholders to push back
+  if it is made prematurely.
+  - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+    a *summary comment* trying to lay out the current state of the discussion
+    and major tradeoffs/points of disagreement.
+  - Before actually entering FCP, *all* members of the working group must sign off;
+  this is often the point at which many working group members first review the RFC
+  in full depth.
+- The FCP lasts ten calendar days, so that it is open for at least 5 business
+  days. It is also advertised widely,
+  e.g. in the [Weekly IPFS All Hands Call](https://github.com/ipfs/pm/#weekly-all-hands). This way all
+  stakeholders have a chance to lodge any final objections before a decision
+  is reached.
+- In most cases, the FCP period is quiet, and the RFC is either merged or
+  closed. However, sometimes substantial new arguments or ideas are raised,
+  the FCP is canceled, and the RFC goes back into development mode.
+
+### Status: Accepted    
+
+Once an RFC becomes "Accepted" then authors may implement it and submit the
```

Comment:
> submit the feature as a pull request

This refers to creating **a new** PR made to a specific repository? Or the one created in this repository for the draft? In that case, it just has to be merged as it was created in the draft stage already.

Reply:
"submit the feature as a pull request to the corresponding repo" seems pretty clear to me. I take it to mean that once the RFC is Accepted you can make corresponding changes to the necessary repositories and code bases by submitting PRs. Can you suggest a change that makes it clearer?